src/app/components/BalanceBox/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { useAccount } from "~/app/context/AccountContext";
+
+function BalanceBox() {
```

Comment:
If I want to use this component for the options page and add the className property `className` to the element I get the following lint error:

`Property 'className' does not exist on type 'IntrinsicAttributes'.`

I found this solves the problem:

```
type Props = {
  className: string;
};

function BalanceBox(props: Props) {
```

Maybe there's a better way to do it, I'm fairly new to React.